File: Admin_p/Admin_p.Web/Modules/electroDB/Orders/OrdersColumns.cs
using Serenity.ComponentModel;
using System;
using System.ComponentModel;

namespace Admin_p.electroDB.Columns;

[ColumnsScript("electroDB.Orders")]
[BasedOnRow(typeof(OrdersRow), CheckNames = true)]
public class OrdersColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int OrderId { get; set; }
    public string UserName { get; set; }
    public DateTime OrderDate { get; set; }
    public decimal TotalAmount { get; set; }
    [EditLink]
    public string Status { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
}

File: Admin_p/Admin_p.Web/Modules/ServerTypes/electroDB/OrdersColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { OrdersRow } from "./OrdersRow";

export interface OrdersColumns {
    OrderId: Column<OrdersRow>;
    UserName: Column<OrdersRow>;
    OrderDate: Column<OrdersRow>;
    TotalAmount: Column<OrdersRow>;
    Status: Column<OrdersRow>;
    CreatedAt: Column<OrdersRow>;
    UpdatedAt: Column<OrdersRow>;
}

export class OrdersColumns extends ColumnsBase<OrdersRow> {
    static readonly columnsKey = 'electroDB.Orders';
    static readonly Fields = fieldsProxy<OrdersColumns>();
}